docs/source/user_guide/automl.ipynb

```diff
@@ -383,8 +453,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# saving the entire automl search\n",
```

Comment:
Isn't Angela's native pickle ready to go?

Reply:
Yes! Native pipeline pickles are good. Updating the docs with that info is tracked by #2158. And verifying that `AutoMLSearch` instances themselves are native-pickleable is tracked by #2174.